XS✔ ◾ Updated outdated image

ozairashfaqueSSW opened the pull request:
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)
(checked by William)
Hi @william-liebenberg,
As per our conversation, you agreed that the new image is great.
Email: SSW Rules - improving rules :
•	The rule “Login Security - Do you know the appropriate error message for an invalid username or password entry?” has an outdated screenshot and it needs to be replaced. See:
[https://www.ssw.com.au/rules/login-security-do-you-know-the-correct-error-message-for-an-incorrect-user-name-or-password/](https://aus01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.ssw.com.au%2Frules%2Flogin-security-do-you-know-the-correct-error-message-for-an-incorrect-user-name-or-password%2F&data=05%7C01%7COzairAshfaque%40ssw.com.au%7C40bcb50c648a489fd8b908dbd99d075b%7Cac2f7c34b93548e9abdc11e5d4fcb2b0%7C0%7C0%7C638343040685183081%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=S3Yypl%2BRporxuCSdQT7ecFDFO2PXwfHzIuoA6j2pqjA%3D&reserved=0)

> 2. What was changed?

- Updated outdated image
New image:
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/96504411/7a158153-0aa0-4fda-8ba8-31577de9d16e)
**Figure: Sugarlearning login screen with appropriate error message**

> 3. Did you do pair or mob programming?

- No, I did not have pair programming, as no code was involved in this PR. 
I had a discussion with @adamcogan and the screenshot in the rule is outdated.
So, I have updated the new image.
<!-- E.g. I worked with @gordonbeeming and @sethdailyssw -->

<!-- 
Check out the relevant rules
- https://www.ssw.com.au/rules/rules-to-better-pull-requests
- https://www.ssw.com.au/rules/write-a-good-pull-request
- https://www.ssw.com.au/rules/over-the-shoulder-prs 
- https://www.ssw.com.au/rules/do-you-use-co-creation-patterns
-->

tiagov8 reviewed (changes requested):
Hi @ozairashfaqueSSW 

I see the new image path, but not the new image uploaded...

- [ ] 1. Please include the image upload in your PR (or link to another PR if it's the case)

ozairashfaqueSSW commented:
> Hi @ozairashfaqueSSW
> 
> I see the new image path, but not the new image uploaded...
> 
> * [x]  1. Please include the image upload in your PR (or link to another PR if it's the case)

I have added image to assets folder and fixed the path

william-liebenberg reviewed (commented):
LGTM
